envoy/datadog_checks/envoy/parser.py

```diff
@@ -92,17 +96,16 @@ def parse_metric(metric, metric_mapping=METRIC_TREE):
     return METRIC_PREFIX + metric, tags, METRICS[metric]['method']
 
 
-def construct_tags(tag_builder, num_tags):
+def construct_tag_values(tag_builder, num_tags):
```

Comment:
Could maybe reformat `num_tags` here to be consistent with the `tags_to_build` reformatting?